subsys/net/lib/sockets/Kconfig

```diff
@@ -108,6 +108,12 @@ config NET_SOCKETS_PACKET
 	  while sending. While receiving, packets (including all the headers)
 	  will be feed to sockets as it as from the driver.
 
+config NET_SOCKETS_CAN
+	bool "Enable socket CAN support [EXPERIMENTAL]"
+	default n
```

Comment:
@ulfalizer asked to not use "default n", which is default itself.

Reply:
Yes, this can be fixed.